src/main/resources/reference.conf

```diff
@@ -1354,6 +1354,106 @@ resourceTypes = {
     }
     reuseIds = false
   }
+  mct_mapping = {
+    actionPatterns = {
+        delete = {
+            description = "delete this resource"
+        }
+        read_policies = {
+            description = "view all policies and policy details for this resource"
+          }
+          set_parent = {
+            description = "set the parent workspace of this resource"
+          }
+          "share_policy::owner" = {
+            description = "change the membership of the owner policy for this resource"
+          }
+          "share_policy::editor" = {
+            description = "change the membership of the editor policy for this resource"
+          }
+          "share_policy::writer" = {
+            description = "change the membership of the writer policy for this resource"
+          }
+          "share_policy::reader" = {
+            description = "change the membership of the reader policy for this resource"
+          }
+          "read" = {
+            description = "perform reader actions on the resource"
+          }
+          "write" = {
+            description = "perform writer actions on the resource"
+          }
+          "edit" = {
+            description = "perform editor actions on the resource"
+          }
+    }
+    ownerRoleName = "owner"
+    roles = {
+      owner = {
+        roleActions = ["delete", "read_policies", "share_policy::owner", "share_policy::editor", "share_policy::writer", "share_policy::reader", "own", "edit", "read", "write", "set_parent"]
```

Comment:
I don't see an `own` action defined in the `actionPatterns`

Reply:
added, thanks